helm/etcd-backup-chart/templates/cronjob.yaml

```diff
@@ -11,6 +11,7 @@ spec:
   jobTemplate:
     spec:
       # Job timeout
+      ttlSecondsAfterFinished: 21600
```

Comment:
Why 21600?

Reply:
This is 6h, so if we create a new backup jobs every 6h we can get rid of failed previous jobs.
If the new backup fails we will have an alarm for it during next 6 hours, if not, we have a more recent backup

Reply:
Okay - could we have a comment for that?

Reply:
makes sense